source/renderer/app/themes/daedalus/light-blue.js

```diff
@@ -594,6 +599,9 @@ export const LIGHT_BLUE_THEME_CONFIG = {
     '--theme-staking-stake-pool-tooltip-text-color': '#5e6066',
     '--theme-staking-stake-pools-search-button-color': '#5e6066',
     '--theme-staking-stake-pools-search-icon-color': 'rgba(94, 96, 102, 1)',
+    '--theme-staking-stake-pools-search-clear-button-background-color':
```

Comment:
@daniloprates please include all newly created variables also in `utils/crateTheme.js`